Dockerfile

```
@@ -7,6 +7,7 @@ ARG RENODE_VERSION=1.13.0
 ARG LLVM_VERSION=12
 ARG BSIM_VERSION=v1.0.3
 ARG WGET_ARGS="-q --show-progress --progress=bar:force:noscroll --no-check-certificate"
+ARG SPARSE_COMMIT=9212270048c3bd23f56c20a83d4f89b870b2b26e
```

to make it consistent with the rest

```suggestion
ARG LLVM_VERSION=12
ARG BSIM_VERSION=v1.0.3
ARG SPARSE_VERSION=v0.6.4
ARG WGET_ARGS="-q --show-progress --progress=bar:force:noscroll --no-check-certificate"
```